server/komodo-rest/src/main/java/org/komodo/rest/relational/response/RestVdb.java

```diff
@@ -150,7 +160,7 @@ public void setDescription(String description) {
      */
     public String getOriginalFilePath() {
         Object path = tuples.get(FILE_PATH_LABEL);
-        return path != null ? path.toString() : null;
+        return path != null ? path.toString() : PATH_DEFAULT;
     }
```

Comment:
Not sure we want a path value of "path" in the rest json. It just adds noise and data to the network transfer.